eng-paths/sr_staff_engineer.md

```diff
@@ -85,9 +87,11 @@ Sr Staff Engineer
 </td></tr>
 <tr><td markdown="1" style="font-weight:bold">
 
-- Technical expert in all key components and solid on the entire codebase.
+- Technical expert in most key components and solid on 2 products.
```

Comment:
I don't like that change, I think the bar should be the entire codebase. Ricardo might be an exception to the rule given his history in devops over code, and Michael may be a little rusty in React, but h makes up for it being the Django authority in staff meetings. I think the bar should be: Solid on the entire codebase.

Reply:
omg totally, thank you for flagging @partner0, this wasn't intentional but rather a copy action done at the wrong line 🤦🏽‍♀️ this is currently duplicated on the sr staff and SE3, but belongs only to SE3. I'll roll it back, 100% aligned, apologies + thanks for catching it!